user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Basic_controls/DropdownFeed.md

```diff
@@ -16,13 +16,13 @@ To configure the component:
 1. Apply the necessary data feeds. See [Applying a data feed](xref:Apply_Data_Feed).
 
    > [!NOTE]
-   > From DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4 onwards<!--RN 38811-->, when you apply a single query data feed, the resulting rows from that query are listed instead of the query itself. Additionally, if you want to use this dropdown feed component as a data feed, the entry will be listed as *Query rows* in the *Data* pane. When you apply multiple query data feeds, the queries themselves are listed as data. Prior to DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4, when you apply a single query data feed, the query itself is listed as data. If you use this dropdown feed component as a data feed, the entry will be listed as *Queries* in the *Data* pane.
+   > From DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4 onwards<!--RN 38811-->, when you apply a single query data source or *Tables* component data source (compatible from DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12 onwards<!--RN 41161-->), individual rows from that query or table are listed instead of the query or table itself. Additionally, if you want to use this dropdown feed component as a data feed, the entry will be listed as *Tables* or *Query rows* in the *Data* pane (depending on your DataMiner version<!--RN 41075-->). When you apply multiple query data sources or *Tables* component data sources (compatible from DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12 onwards<!--RN 41161-->), the queries or tables themselves are listed as data. Prior to DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4, when you apply a single query data feed, the query itself is listed as data. If you use this dropdown feed component as a data feed, the entry will be listed as *Queries* in the *Data* pane.
```

Comment:
It's not only the Tables from **components**, but also variables or flows that expose a table. The same goes for every spot where tables can be used. There are basically multiple spots that expose data;
- Components
- Flows
- Url
- Variables
- ...
All data they expose are typed (e.g., element, table, number), and can be used everywhere where that type is supported, regardless what the origin of the data is.

Reply:
```suggestion
   > From DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4 onwards<!--RN 38811-->, when you apply a single query data source or *Tables* data source (compatible from DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12 onwards<!--RN 41161-->), individual rows from that query or table are listed instead of the query or table itself. Additionally, if you want to use this dropdown feed component as a data feed, the entry will be listed as *Tables* or *Query rows* in the *Data* pane (depending on your DataMiner version<!--RN 41075-->). When you apply multiple query data sources or *Tables* data sources (compatible from DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12 onwards<!--RN 41161-->), the queries or tables themselves are listed as data. Prior to DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4, when you apply a single query data feed, the query itself is listed as data. If you use this dropdown feed component as a data feed, the entry will be listed as *Queries* in the *Data* pane.
```

@GillesBara, is this better?